docs/guide-using-an-ide-for-backend.md

```diff
@@ -161,9 +160,25 @@ Sadly, it's not possible to generate multiple files from one template, so the Me
 * Run, Edit configurations..., `+`, Tomcat, Local.
 * Call it `molgenis-app [exploded]`
 * (Add and) select your Tomcat installation
-* VM options: `-Dmolgenis.home=<path to dir containing molgenis-server.properties> -Xmx4g -Des.discovery.zen.ping.multicast.enabled=false -Des.network.host=localhost`
+* Select the 'Startup/Connection' tab
+  Copy and paste these variables in the Environment variables area and select 'Pass environment variables' 
+  ```properties
+  molgenis.home=**your own version of the data dir**
+  opencpu.uri.host=localhost
+  elasticsearch.transport.addresses=localhost:9300
+  db_uri=jdbc:postgresql://localhost/molgenis
+  db_user=molgenis
+  db_password=molgenis
+  admin.password=admin
+  MINIO_BUCKET_NAME=molgenis
+  MINIO_ENDPOINT=http://localhost:9000
+  MINIO_ACCESS_KEY=molgenis
+  MINIO_SECRET_KEY=molgenis
```

Comment:
How can we access the minio data folder now? (This conflicts with the way we wanted to do the backups).

Reply:
is now mounted in ~/.molgenis/minio